src/setup/cortx_setup.py

```diff
@@ -64,8 +80,21 @@ def add_args(parser: str):
         parser.add_argument('node_id', help='node_id')
         parser.add_argument('cortx_conf', nargs='?', help='CORTX Config URL')
 
+    def _validate(self):
+        """ Validate cluster command args """
+
+        if self._args.action not in ['bootstrap']:
+            raise CortxProvisionerError(errno.EINVAL, 'Invalid action type')
+
+        path = urlparse(self._args.cortx_conf).path
```

Comment:
URL need not be a file path. It can be consul URL as well in which case this path will not exist. Need to remove this

Reply:
Removed this validation.

Reply:
Done.